netlify.toml

```diff
@@ -10,5 +10,5 @@
 
 [[redirects]]
   from = "https://docs.openservicemesh.io/*"
-  to = "https://release-v1-0.docs.openservicemesh.io/:splat"
+  to = "https://release-v1-1.docs.openservicemesh.io/:splat"
```

Comment:
This change won't do anything on the release branch I don't think. I think this only has any effect when done on `main`.

Reply:
Should I delete and make new PR for main?

Reply:
You can make the change in main once this PR is complete and you have created `https://release-v1-1.docs.openservicemesh.io`. Bridget can help with that once you are ready.

Reply:
I **think** I did that already #349  - should I do it again?

Reply:
Awesome, nope! Didn't realize you had already done that. This change is fine to make on the release branch, but it must also be made on main for `https://docs.openservicemesh.io/` to redirect to `https://release-v1-1.docs.openservicemesh.io`

Reply:
#355  exists now 🙏🏻